README.md

```diff
@@ -12,10 +12,14 @@ This is a custom fork of Open edX's docker based development environment, frankl
 
 ## Environment configuration
 
-The `OPENEDX_RELEASE` variable should be set in your \*nix environment. Use the following command to set it.
+The `OPENEDX_RELEASE` variable should be set in your \*nix environment. Use the following command to set it. Currently, `ironwood.master` and `hawthorn.master` releases of Open edX are supported.
 
 ```
-$ export OPENEDX_RELEASE=ironwood.master
+export OPENEDX_RELEASE=ironwood.master
```

Comment:
I would suggest pinning the `OPENEDX_RELEASE` to named release tags instead of just pointing the head to master. https://edx.readthedocs.io/projects/edx-developer-docs/en/latest/named_releases.html#ironwood

Reply:
Well, that isn't how the devstack processes Open edX repos. It needs a dedicated branch to be able to properly clone and checkout repos.